source/extensions/extensions_build_config.bzl

```diff
@@ -269,6 +269,15 @@ EXTENSIONS = {
     #
 
     "envoy.http.stateful_header_formatters.preserve_case":       "//source/extensions/http/header_formatters/preserve_case:preserve_case_formatter",
+
+    #
+    # Original IP detection
+    #
+
+    # Disabled by default
```

Comment:
Is this true? How is it disabled by default?

Reply:
I think it is, via:

```
envoy_extension_package(enabled_default = False)
```

in source/extensions/http/original_ip_detection/custom_header/BUILD...

Reply:
kill_request does the same thing in that same file, in the HTTP filters section:

```
    # Disabled by default
    "envoy.filters.http.kill_request":                  "//source/extensions/filters/http/kill_request:kill_request_config",
    ```

Reply:
I would just enable this one by default?

Reply:
I was inclined to enabling too, Alyssa was more on the no side. But now we can overrule her 😁 

It shouldn't bloat the binary by that much, but I do want to be mindful...

Reply:
for the record, I only wanted the new custom header off by default - original ip has always been part of the binary so IMO that should be in by default.  Happy to be overridden on the former though :-P

Reply:
Hold on, there's some confusion here:

```
+    # Disabled by default
+    "envoy.http.original_ip_detection.custom_header":        "//source/extensions/http/original_ip_detection/custom_header:config",
+    "envoy.http.original_ip_detection.xff":                  "//source/extensions/http/original_ip_detection/xff:config",
```

The comment only refers to the extension immediately below it, not everything below it (because of course we need the XFF extension).

@mattklein123 were you talking about the custom header extension (disabled) or the XFF extension (enabled) or both?

Reply:
I'm talking about the disabled custom header extension. We don't do this for any other extensions (other than kill request which is a special case IMO) so I'm not sure why we would do it for this one? People that do custom builds can leave it out?

Reply:
I have a very light preference for leaving this out given that honestly that extension is mostly there for educational purposes on writing extensions, not sure how much it'll be used (if at all).

Reply:
Actually it was @antoniovicente's idea to refactor it from tests and include it as an example.

Reply:
If it's not in the build I don't think it should be in the docs then? Can you figure out how to exclude it?

Reply:
NP -- excluding it via `[#not-implemented-hide:]` (thanks @phlax).